src/main/java/com/epam/aidial/core/data/Bucket.java

```diff
@@ -1,4 +1,4 @@
 package com.epam.aidial.core.data;
 
-public record Bucket(String bucket) {
+public record Bucket(String bucket, String appdata) {
```

Comment:
it's better not to serialize null values for bucket object, consider adding @JsonInclude(NOT_NULL)

Reply:
fixed